ecosystem/sep-0031.md

```diff
@@ -857,9 +853,16 @@ In certain cases the Receiving Anchor might need to request updated information
 the bank tells the Receiving Anchor that the provided Receiving Client's name is incorrect or missing a middle initial.
 Since this information was sent via SEP-12, the transaction should go into the `pending_customer_info_update` state
 until the Sending Anchor makes another `SEP-12 PUT /customer` request to update. The Sending Anchor can check which
-fields need to be updated by making a `SEP-12 GET /customer` request including the `id` or `account` & `memo`
-parameters. The Receiving Anchor should respond with a `NEEDS_INFO` status and `last_name` included in the fields
-described.
+fields need to be updated by making a `SEP-12 GET /customer` request. The Receiving Anchor should respond with a
+`NEEDS_INFO` status, a `provided_fields` including all provided fields, and a `fields` including all fields need to be
+updated .
+
+1. `GET /transaction` and verify transaction status is `pending_customer_info_update`
+2. Get required fields via [SEP-12] `GET /customer` request. It is recommended to use the combination of
+   `transaction_id` and customer type `type` (`sep31-sender` and `sep31-receiver`) as part of the request.
```

Comment:
Should we mention `sep31-sender` and `sep31-receiver` here? It looks like these two types are the only acceptable types.

Reply:
Will remove